2019-Q4.md

```diff
@@ -0,0 +1,166 @@
+
+
+### Company key result: 3 clients PoC started
+
+Initiative: Hire solutions Engineer and further refine process to kick off PoC promptly 
+
+Initiative: Deliver Technical documentation & guides for Enterprise implementation and onboarding (i.e for different infrastructure, use cases, tech stacks)
+
+
+### Company key result: $3M new revenue in top of the funnel
+
+Initiative: Deliver more sales collaterals including top engineering metrics, Ebooks and datasheets 
+
+Initiative: Refocus Inbound lead generation efforts towards higher Enterprise profiles to increase % of requests from executives
+
+Initiative: Develop, execute and measure success of our Outbound efforts
+
+Initiative: Effectively support product releases with integrated campaigns 
+
+
+### Company key result: 3M revenue moved into proposal stage
+
+Initiative: Refine strategy and processes from MQA outreach to actual workshop delivery (shared with sales)
+
+Initiative: Plan and develop Enterprise workshop materials
+
+Initiative: Hire a pre-sales Engineer (shared with talent)
+
+Initiative: Develop pre-sales materials such RFP response, Datasheets, Objection Handling, the anatomy of a deal, demo scripts, Competitive teardowns & Technical roadmap (shared with sales)
+
+
+### Company key result: Getting user traction for source{d} CE
+
+Initiative: Setup the feedback loop with source{d{} users
+
+Initiative: Improve onboarding user experience 
+
+
+## Company objective: Deliver a unified product experience
+
+
+### Company key result: Bring query results under 10s for datasets of 10k repos
+
+Initiative: TBD
+
+Initiative: TBD
+
+
+### Company key result: Add 7 data sources
+
+Initiative: TBD
+
+Initiative: TBD
+
+
+### Company key result: Deliver out-of-the-box dashboards
+
+Initiative: TBD
+
+Initiative: TBD
+
+
+## Company objective: Be a better company to work at
+
+
+### Company key result: Tackle 60% of issues raised in Candidate Feedback survey
+
+Initiative: Take back the responses seen the last time we ran the survey
+
+Initiative: Discuss with Robbie what issues still stand
+
+Initiative: Deliver a plan to tackle the main issues raised by candidates
+
+
+### Company key result: Improve candidate experience during the hiring process
+
+Initiative: Take back the responses seen the last time we ran the survey
+
+Initiative: Discuss with Robbie what issues still stand
+
+Initiative: Deliver a plan to tackle the main issues raised by candidates
+
+
+### Company key result: Tackle 60% of issues raised in Onboarding Feedback
+
+Initiative: Meet with Talent to understand the issues raised on Onboarding Feedback
+
+Initiative: Deliver an action plan to tackle main issues raised on Onboarding Feedback
+
+
+### Company key result: Tackle 60% of issues raised in Employee Engagement survey
+
+Initiative: Meet with Management to identify the sources of problem
+
+Initiative: Deliver an action plan to tackle main issues raised last quarter survey
+
+
+# Team OKRs
```

Comment:
As is Company and Team OKRs seem redundant to me. "Deliver a unified product experience" is both a company and a team objective with a very different set of initiatives under. Should we all have teams OKRs ? In the case of Marketing / Sales, they would be identical to Company ones.

Reply:
Yes, all teams involved on Company OKRs should have their Team OKRs, but to be honest we didn't have that much of an engagement with the OKRs this quarter, although I tried asking to update. The way out I found was to push the PR and see if we get more movement from here.